src/jlib.erl

```diff
@@ -337,17 +331,17 @@ rsm_decode(#xmlel{} = SubEl) ->
     end.
 
 -spec rsm_parse_element(exml:element(), rsm_in()) -> rsm_in().
-rsm_parse_element(#xmlel{name = <<"max">>, attrs = []} = Elem, RsmIn) ->
+rsm_parse_element(#xmlel{name = <<"max">>, attrs = #{}} = Elem, RsmIn) ->
```

Comment:
Actually in this case I'm not sure if the XEP requires the list of attributes to be empty, but, if that's not the case, we can just remove all the `attrs = #{}` from the clauses, they're doing nothing else than failing if the `attrs` was not a map.